### ISSUE TITLE
Is it possible to send DS.Model to modal component ?

### ISSUE THREAD

**Issue by Meph-** [ef4/liquid-fire#260]
I have no luck with:  

{{#each item in model}}
...
{{#link-to (query-params item=item) tagName="button"}}
                    <label>{{item.title}}</label>
                {{/link-to}}

export default Ember.ArrayController.extend({
    queryParams:['item'],
   item: null,
...
});

template:
<div>Change Title</div>
<div>{{input value=item.title}}</div>
<button {{action "gotIt"}} class="done">Thanks</button>
<button {{action "change"}} class="change">Change</button>

**Comment by ef4** [comment]
Yes, it should be possible to send a DS.Model. I don't see anything wrong with your example.

I would need more info to help figure out what's wrong.

**Comment by Meph-** [comment]
Sorry for my late response. When I used code above and I specified "withParams: ['item']"  in router, then  I got item= [object Object] as string value in modal view. It looks like query-params takes only string parameter. When I use otherParams: ['item'] in router, that works fine and I can use item as object in model. Is it possible send object from template?

**Comment by trisrael** [comment]
I ended up having query params watching `itemId` and then passing `withParams: ['item']` where `item` is a computed property in the observer watching `itemId`

**Comment by berdof** [comment]
Have anybody solved this ? I need to pass model to a modal to modify it. Is it actually possible?
